universal-application-tool-0.0.1/app/assets/javascripts/azure_upload.ts

```diff
@@ -72,12 +77,14 @@ class AzureUploadController {
   private setFileUploadMetadata(
     redirectUrl: URL,
     azureUploadProps: any,
-    resp: any
+    resp: any,
+    blockBlobUrl: any
```

Comment:
suggestion (blocking): we miss the benefits of static type checking with the `any` type. Please annotate the narrowest feasible type for these parameters.

Reply:
I deleted this file so I'll resolve all comments related to it.